orbs/phraseapp/utils/autocreate-phraseapp-pr

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# This script creates a pull request on GitHub with updated locales from PhraseApp,
+# bypassing the need for human intervention via PhraseApp "GitHub Sync" feature.
+
+# Use circleci user
+git config --global user.email "dev@jobteaser.com"
```

Comment:
**question**

Knowing that this orb is public, I am wondering if we shouldn't hide which user / mail is used ?

Reply:
This email is used in other files of this repository, but I'll hide it anyway. ;)

Reply:
Oh? Nevermind then 😄.